news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,247 @@
+
+If you thought *My Man* had LNs, wait until you see a map of over 7 minutes of non-stop LN gameplay, and with not just single-note rainbows and glissandos, but an interconnected web of the two. It is an impeccable display of skill, accuracy, speed, and just what osu!monthly is looking for in a showcase!
+
+<iframe width="100%" height="315" src="https://youtu.be/embed/nc_ejGtZKUM" frameborder="0" allowfullscreen></iframe>
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+
+#### Galaxy Brain
+
+Somebody scream! Even with being the number one osu!catch player in the world, a play such as [YesMyDarkness’](https://osu.ppy.sh/users/4158549/fruits) is no small feat. 
+
+Coming in with a HD HR 4xmiss on [Kurokotei’s Galaxy Collapse](https://osu.ppy.sh/beatmapsets/521900#fruits/1108507) one of the most well known artists and songs in osu! surely does do them some justice. Riddled with intense and **dash**ingly hard jumps it is a play to contend with even the hardest of maps, let alone with hidden and hardrock! 
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/byyHOAopo1A" frameborder="0" allowfullscreen></iframe>
+
+---
+
+## New Features on osu!(lazer)!
+
+The new client has never so **close** from releasing (there's still a lot of work on it!), and has **new mods**, more original than never! 
+For the first time ever, lazer patch notes have been posted on the official osu! [YouTube channel](https://www.youtube.com/c/osugame), compiled by [pishifat](https://osu.ppy.sh/users/3178418) with further insight provided by the man himself, [peppy](https://osu.ppy.sh/users/2)!
+Check the video out:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/67P4LKlAh-0" frameborder="0" allowfullscreen></iframe>
+
+If you are interested in all previous and future patch notes, you can find them [here](https://osu.ppy.sh/home/changelog).
+
+One thing you might've missed is the new fancy Repel mod added to lazer, which is pretty chaotic but fun (finally a worthy opponent for [badeu](https://osu.ppy.sh/users/1473890)?)
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://user-images.githubusercontent.com/47010459/172435116-6b09a48c-6fd6-4abb-9e5f-e96ab3565184.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+Another feature that's maybe not the newest, but some of you might have missed, or just haven't gotten around to trying yet is the **skin layout editor**! First implemented in May of 2021, this is still very much a work-in-progress feature, however currently you are able to edit the placement of all skin elements that appear during gameplay! Check out the showcase video from the original patch notes below:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://user-images.githubusercontent.com/191335/118155608-5c5fce80-b453-11eb-88f4-e552b10ed5f1.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## Summary of this Article
```

Comment:
on further thinking, we should do one of a few things:

1) remove the summary section entirely, since i don't expect it to get this much traffic this far inside the article as it stands
2) keep it, but condense it down significantly (mostly by changing the "article here" links to direct references to newsposts instead of sections inside the article)
3) as 2, but shift the section up to be the first actual content section under the table of contents so that it is encountered first

Reply:
Guess we'll remove it.